### ISSUE TITLE
[Desktop] There was a problem processing your contribution message on BAT logo

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#10329]
## Description 

There was a problem processing your contribution message on BAT logo

## Steps to Reproduce

   1. Clean profile 1.11.x staging environment
   2. Launch brave using `brave.exe --enable-logging --vmodule="*/bat-native-ads/*"=6,"*/bat-native-confirmations/*"=6,"*/bat-native-ledger/*"=6 --rewards=staging=true,reconcile-interval=3,short-retries=true`
   3. Enable rewards
   4. Connect KYC'd uphold staging account ( should have some balance (self funded or ad grants))
   5. Add connected pub DDG to AC table
   6. Wait for AC to trigger
   8. Click on BAT logo displays the error message as `There was a problem processing your contribution`
   9. Dismiss the notification.
  10. Click on BAT logo again shows the auto contribution successful message as expected

**Note:**  This issue is happening on Android also (followed same steps, except step 2)

## Actual result:

There was a problem processing your request message on BAT logo
![image](https://user-images.githubusercontent.com/38657976/84876029-60009f00-b0a4-11ea-9ce4-3b7b88e3048e.png)

**Android 1.10.92**
![Screenshot_2020-07-17-13-59-27](https://user-images.githubusercontent.com/38657976/84876973-85da7380-b0a5-11ea-8136-a6c4de65028b.png)

## Expected result:

`There was a problem processing your contribution message` should not be shown on BAT logo

## Reproduces how often: 

Always

## Brave version (brave://version info)

Brave | 1.11.65 Chromium: 83.0.4103.97 (Official Build) dev (64-bit)
-- | --
Revision | 326d148b9655369b86498d9ecca39f63dd2bdd2d-refs/branch-heads/4103@{#657}
OS | Windows 10 OS Version 1903 (Build 18362.30)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @NejcZdovc

**Comment by NejcZdovc** [comment]
@GeetaSarvadnya can you please attach logs from the client side

**Comment by GeetaSarvadnya** [comment]
Provided logs via DM for privacy reasons

**Comment by emerick** [comment]
Related to / blocked by https://github.com/brave/brave-core/pull/5883.

**Comment by kjozwiak** [comment]
@GeetaSarvadnya mind running through this again once we get a new `1.11.x` build now that https://github.com/brave/brave-core/pull/5901 as been merged and see if you're still experiencing the same issue?

**Comment by GeetaSarvadnya** [comment]
@kjozwiak I am not able to reproduce the issue in `1.11.92` latest dev build. Click on BAT logo is showing the correct AC notification with AC amount there is no error message, looks like the issue is fixed by the PR #5901

![image](https://user-images.githubusercontent.com/38657976/87009630-0e4ebe80-c1e3-11ea-9d0c-dc03449ddbb3.png)

**Comment by GeetaSarvadnya** [comment]
@srirambv Please test the issue on Android and see whether the issue is fixed or not, if not open a new issue in Android.
